validation-test/Evolution/test_backward_deploy_struct.swift

```diff
@@ -0,0 +1,52 @@
+// RUN: %target-resilience-test --backward-deployment
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import backward_deploy_struct
+
+
+var BackwardDeployStructTest = TestSuite("BackwardDeployStruct")
+
+BackwardDeployStructTest.test("ResilientStruct") {
+  if getVersion() == 1 {
+    // FIXME: IRGen inserts the metadata load outside the version check
+    // apparently. Work around that here. <rdar://problem/46438608>
```

Comment:
I think it respects `#available` but not arbitrary weak linking. Maybe put in a dummy `#available` for now?

Reply:
I'm going to skip this one for now.